### PR TITLE
Adds Research Security

### PR DESCRIPTION
**Pull request by XanderDox**
## About The Pull Request

Adds a new job to science, 'Research Security' or alt titled 'Containment Specialist'. This position is subservient to the Director and all other Science staff, and responsible for protecting Science's assets, and assisting in keeping all scientific products within science.

## Why It's Good For The Game

The position has been requested by some of our science mains, specifically a prolific RD main. A lot of science recently has been dedicated to the capture of dangerous creatures and their study, and having the Director use law enforcement do this looks strange and takes police away from dealing with crime/griefers.

Additionally, Science has always had a security kiosk but nobody to man it. The result of a lack of security presence has meant that museum gets robbed constantly, and science is rarely secure despite being one of the locations that should be the most secure on the map.

There has been an argument about them 'acting like police'. I don't really see this as a concern, and City Hall Security in the last few weeks has improved a lot in terms of not acting like pseudo-cops, only really acting when there's griefers. I think we should trust the players and at least trial the role.

## Changelog
:cl:
add: Geminus Laboratories has contracted a private security guard.
/:cl:

### CODE REVIEW

**Comment by XanderDox** [comment]
PS this PR removes City Hall Security from the category police are filed under on the manifest, this should help with any confusions.

**Comment by XanderDox** [comment]
PS this PR contains no mapping updates, as I am going to recruit @WSPaean to handle them if this is approved by @GeneriedJenelle

**Comment by XanderDox** [comment]
There is now a merge conflict. I'll resolve it shortly.

File: code/game/jobs/job/police.dm [hunk 1]
```diff
@@ -140,8 +140,8 @@
 	title = "City Hall Guard"
 	email_domain = "secure.plux.gov.nt"
 	flag = BRIDGE
-	department = DEPT_COUNCIL
-	department_flag = ENGSEC
+	department = "Civilian"
```
Comment (GeneriedJenelle) [inline code/game/jobs/job/police.dm]: We're using the new department system now, don't use "Civilian" use DEPT_PUBLIC, DEPT_RESEARCH, or DEPT_POLICE, etc. See department_presets.dm for list of them.

File: code/game/jobs/job/science.dm [hunk 2]
```diff
@@ -114,3 +114,26 @@
 	minimal_player_age = 0
 
 	outfit_type = /decl/hierarchy/outfit/job/science/intern
+
+/datum/job/rguard
+	title = "Research Security"
+	email_domain = "sciworks.nt"
+	flag = SCIGUARD
+	department = "Research and Science"
```
Comment (GeneriedJenelle) [inline code/game/jobs/job/science.dm]: DEPT_RESEARCH, here.

File: code/game/jobs/job/police.dm (hunk 1 above)
Comment (XanderDox) [reply code/game/jobs/job/police.dm]: Roger.

**Comment by XanderDox** [comment]
> Update the department settings and I'll merge this in.

All done!

**Comment by XanderDox** [comment]
See #943 for the mapping component to this.

**Pull request merged**